.editorconfig

```diff
@@ -1,18 +1,17 @@
 # EditorConfig is awesome: http://EditorConfig.org
 
-# Howto with your editor:
-#   Sublime: https://github.com/sindresorhus/editorconfig-sublime
+# How-to with your editor: http://editorconfig.org/#download
 
 # top-most EditorConfig file
 root = true
 
 # Unix-style newlines with a newline ending every file
-[**]
+[*]
```

Comment:
When no `/` is present in the glob, it will apply to files in all subdirectories, so `**` would be unecessary.